test/evergreen.yml

```diff
@@ -3008,6 +3036,105 @@ tasks:
 
             # Run S3 extension unit testing.
             ext/storage_sources/s3_store/test/run_s3_unit_tests
+  - name: azure-tiered-unittest-test
+    tags: ["pull_request", "tiered_unittest"]
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          <<: *configure_flags_tiered_storage_azure
+          HAVE_UNITTEST: -DHAVE_UNITTEST=1
+          cmake_binary: /opt/mongodbtoolchain/v4/bin/cmake
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/cmake_build"
+          shell: bash
+          include_expansions_in_env:
+            - azure_sdk_ext_access_key
+          script: |
+            set -o errexit
+            # Set the Azure credentials using config variable.
+            export AZURE_STORAGE_CONNECTION_STRING="${azure_sdk_ext_access_key}"
+            set -o verbose
+
+            # Run AZURE extension unit testing.
```

Comment:
This reference should be title-case, i.e. `Azure`.

Reply:
Done! [5f6589f](https://github.com/wiredtiger/wiredtiger/pull/8842/commits/5f6589fa932fdb03b4dc0351fae386913ae89d4e)